src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -967,6 +938,39 @@ class LicenceService(
         )
       )
     )
+
+    val sentenceChanges = getSentenceChanges(sentenceDatesRequest, licenceEntity)
+
+    log.info(
+      "Date change flags: LSD ${sentenceChanges.lsdChanged} LED ${sentenceChanges.ledChanged} " +
+        "SED ${sentenceChanges.sedChanged} TUSSD ${sentenceChanges.tussdChanged} TUSED ${sentenceChanges.tusedChanged} " +
+        "isMaterial ${sentenceChanges.isMaterial}"
+    )
+
+    if (!sentenceChanges.isMaterial) return
+
+    // Notify the COM of any change to material dates on the licence
+    updatedLicenceEntity.bookingId?.let {
+      prisonApiClient.hdcStatus(it)
+        .defaultIfEmpty(PrisonerHdcStatus(passed = false, approvalStatus = "UNKNOWN"))
+        .filter { h -> h.approvalStatus !== "APPROVED" }.subscribe {
+          log.info("Notifying COM ${licenceEntity.responsibleCom?.email} of date change event for $licenceId")
+          notifyService.sendDatesChangedEmail(
+            licenceId.toString(),
+            licenceEntity.responsibleCom?.email,
+            "${licenceEntity.responsibleCom?.firstName} ${licenceEntity.responsibleCom?.lastName}",
+            "${licenceEntity.forename} ${licenceEntity.surname}",
+            licenceEntity.crn,
+            mapOf(
+              Pair("Release date", sentenceChanges.lsdChanged),
+              Pair("Licence end date", sentenceChanges.ledChanged),
+              Pair("Sentence end date", sentenceChanges.sedChanged),
+              Pair("Top up supervision start date", sentenceChanges.tussdChanged),
+              Pair("Top up supervision end date", sentenceChanges.tusedChanged),
```

Comment:
there's an infix method `to` you can use to create pairs:

`"Top up supervision end date" to sentenceChanges.tusedChanged`